check-matrix.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+for workflow in "${GITHUB_WORKSPACE}"/.github/workflows/*.yml; do
```

Comment:
When I run this locally, I get:
```
grep: /.github/workflows/*.yml: No such file or directory
```
because `GITHUB_WORKSPACE` is undefined in my local env, and so we have a preceding slash. Can we make this work locally, too?

Reply:
Ah good catch, will fix.

Reply:
Done, PTAL.